src/Document/UseElement.ts

```diff
@@ -73,6 +73,13 @@ export default class UseElement extends RenderedElement {
 					element.getAttribute('overflow').getString()
 				);
 				tempSvg.children = element.children;
+
+				// element is still the parent of the children
+				element.styles.opacity = new Property(
```

Comment:
```suggestion
				tempSvg.styles.opacity = new Property(
```

Reply:
I don't think this change is possible. The way `opacity` works is like any other style where it crawls up the `.parent` tree. In this case, the parent of the childrens` elements (i.e. `element.children[0].parent`) is still `element`, not `tempSvg` so when they render, the `element` has to have the right opacity for them to pick it up (hence my comment).

In an earlier commit, I tried `element.children.forEach(c => c.parent = tempSvg)` but that broke other places. Maybe we need a `virtualParent` concept in the future because we do mutate the parent and I don't like that in places below:
https://github.com/canvg/canvg/blob/8727b1e5425744f6eeccef3ca668d2b68519c2e6/src/Document/UseElement.ts#L100-L104